galasa-ui/src/components/common/StatusIndicator.tsx

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Another PR was delivered on Friday that includes a "StatusCheck" component, which is very much similar to this "StatusIndicator" component - see https://github.com/galasa-dev/webui/blob/main/galasa-ui/src/components/common/StatusCheck.tsx

It would be good to only have one such component to avoid having multiple components that do the same thing, and I think this StatusIndicator component is easier to read and maintain.